src/libcalamaresui/utils/Paste.cpp

```diff
@@ -40,11 +40,15 @@ logFileContents()
         return QByteArray();
     }
     QFileInfo fi( pasteSourceFile );
-    if ( fi.size() > 16_KiB )
+    sizeLimit = ( sizeLimit < 0 ) ? 1024*1024 : sizeLimit * 1024;            //For KiB to bytes
```

Comment:
Hm .. I was also going to suggest using (from `utils/Units.h`) function `KiBtoBytes()`, but that takes unsigned. You would need to cast `sizeLimit` to unsigned for that (which is ok: in that branch of the ternary, you know it is non-negative)

Reply:
You asked to convert KiB to bytes in `Calamares::Branding::uploadServerFromMap()` for _not having to_ care about units everytime UploadServerInfo object is made. But _utils_ in _libcalamaresui_ do not provide unit converter. So either I have to explicitly convert them (*1024) or call utils from _libcalamares_. I guess 1st one would be better ?

Reply:
`src/libcalamares/utils/Units.h` (due to the way include paths are, just `#include "utils/Units.h"` ), then `CalamaresUtils::Units::KiBtoBytes()`